### PR TITLE
docs: add the paragraph component documentation

### PR DESCRIPTION
**Pull request by SiTaggart**
Copy paste the paragraph component documentation into the doc site

### CODE REVIEW

File: packages/paste-core/components/paragraph/CHANGELOG.md [hunk 1]
```diff
@@ -7,95 +7,24 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
...
 
-
 ### Features
 
-* **icons:** delete old icons, add new streamline icons ([#129](https://github.com/twilio-labs/paste/issues/129)) ([571791d](https://github.com/twilio-labs/paste/commit/571791ded8ee4c55bb5a3dbcebee4b17b2c7c826))
-* **typography:** heading component ([#149](https://github.com/twilio-labs/paste/issues/149)) ([4e033e6](https://github.com/twilio-labs/paste/commit/4e033e6f3e58566107271b18057532e768420827))
-* enable theme switching on the docsite ([#124](https://github.com/twilio-labs/paste/issues/124)) ([df797e5](https://github.com/twilio-labs/paste/commit/df797e57ff1268367053d717f3b3c2ca48a0aa4f))
-
-
-### BREAKING CHANGES
```
Comment (TheSisb) [inline packages/paste-core/components/paragraph/CHANGELOG.md]: :+1:

File: packages/paste-website/src/pages/components/paragraph/index.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+</LivePreview>
+
+## Composition Notes
+
+### When to Use a Paragraph
+
+Paragraphs should be used for most text blocks. Paragraphs provide defaults for font-size, line-height, and margin-bottom. Paragraphs can be used within other components as well. Typically, these are anywhere from 1-5 sentences. Use paragraphs like you would when authoring a word document, to group sentences of a similar theme.
+
+<DoDont>
+  <Do title="Do" body="Use paragraphs for text blocks." center />
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/paragraph/index.mdx]: I really think we should go the extra mile here and prevent merging docs without examples.  It should be pretty straightforward to come up with code examples for these.

File: packages/paste-website/src/pages/components/paragraph/index.mdx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+<Paragraph>
+  Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur praesentium voluptate facere suscipit maiores vitae explicabo error ea? Quisquam, nisi excepturi aut esse non iste dolorum dicta. Tempore, doloribus voluptate!
+</Paragraph>;
+```
+
+#### Props
+
+| Prop      | Type                     | Description | Default |
+| --------- | ------------------------ | ----------- | ------- |
+| children? | {'React Node', 'string'} |             | null    |
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/paragraph/index.mdx]: `{'React Node', 'string'}` can simply be `ReactNode`, which includes string.

File: packages/paste-website/src/pages/components/paragraph/index.mdx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+| line-height   | line-height-30     | No          |
+| font-weight   | font-weight-normal | No          |
+| margin-bottom | space-60           | No          |
+| color         | color-text         | No          |
+
+---
+
+## Usage Guide
+
+### API
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/paragraph/index.mdx]: nit: is this header even necessary?

File: packages/paste-website/src/pages/components/paragraph/index.mdx [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+## Guidelines
+
+By default, the paragraph component should provide consistent values for font-size, line-height, and margin-bottom across themes.
+
+### About Paragraphs
+
+Paragraphs are distinct sections of content, typically dealing with a single theme, and are indicated by a new line. Paragraphs are often used for a majority of text on the page and should be able to support a variety of information-dense layouts.
+
+Paragraph is part of the Typography component, which includes Header and List as well.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/paragraph/index.mdx]: Typography component -> Typography package
Would be nice if we could link to it too (on NPM or docs when we eventually have them).

File: packages/paste-website/src/pages/components/paragraph/index.mdx (hunk 4 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/components/paragraph/index.mdx]: Yeah, it's an odd one, but it's on all the other pages, so consistency? 🤷‍♂

**Pull request merged**